registration/include/pcl/registration/impl/gicp.hpp

```diff
@@ -42,6 +42,7 @@
 #define PCL_REGISTRATION_IMPL_GICP_HPP_
 
 #include <pcl/registration/exceptions.h>
+#include <pcl/registration/impl/icp.hpp>
```

Comment:
I'm not sure if this is ok to do or whether gicp should now also get explicit instantiations.